users/vieting/experiments/switchboard/hybrid/feat/baseline_args.py

```diff
@@ -221,11 +227,19 @@ def get_returnn_config(
                 network["features"]["subnetwork"]["specaug"] = specaug_layer_jingjing(in_layer=["conv_h_act"])
                 network["features"]["subnetwork"]["conv_h_split"]["from"] = "specaug"
                 network["source"] = {"class": "copy", "from": "features"}
+                prolog = get_funcs_jingjing()
             else:
-                network["source"] = specaug_layer_jingjing(in_layer=["features"])
-            prolog = get_funcs_jingjing()
+                if specaug_time_only:
+                    network["source"] = specaug_layer_only_time(in_layer=["features"])
+                    prolog = get_funcs_only_time()
+                else:
+                    network["source"] = specaug_layer_jingjing(in_layer=["features"])
+                    prolog = get_funcs_jingjing()
 
         network = fix_network_for_sparse_output(network)
+    else:
```

Comment:
```suggestion
    else:
        assert not specaug_mask_sorting and not specaug_after_first_layer and not specaug_time_only, "specaug options are specified, but enable_specaug=False"
```
This assertion can help to avoid misunderstandings like by accident switching SpecAugment off even one of those options is desired. Please format it correctly using black or similar, because most likely the line is too long right now.